lib/index.js

```diff
@@ -789,14 +789,16 @@ var powerbi;
                         return 1;
                     }
                     function createScale(options) {
-                        var pixelSpan = options.pixelSpan, dataDomain = options.dataDomain, metaDataColumn = options.metaDataColumn, outerPadding = options.outerPadding || 0, isScalar = !!options.isScalar, isVertical = !!options.isVertical, forcedTickCount = options.forcedTickCount, categoryThickness = options.categoryThickness, shouldClamp = !!options.shouldClamp, maxTickCount = options.maxTickCount;
+                        var pixelSpan = options.pixelSpan, dataDomain = options.dataDomain, metaDataColumn = options.metaDataColumn, outerPadding = options.outerPadding || 0, isScalar = !!options.isScalar, isVertical = !!options.isVertical, forcedTickCount = options.forcedTickCount, categoryThickness = options.categoryThickness, shouldClamp = !!options.shouldClamp, maxTickCount = options.maxTickCount, disableNice = options.disableNice;
                         var dataType = getCategoryValueType(metaDataColumn, isScalar);
                         var maxTicks = isVertical ? getRecommendedNumberOfTicksForYAxis(pixelSpan) : getRecommendedNumberOfTicksForXAxis(pixelSpan);
                         if (maxTickCount &&
                             maxTicks > maxTickCount)
                             maxTicks = maxTickCount;
                         var scalarDomain = dataDomain ? dataDomain.slice() : null;
                         var bestTickCount = maxTicks;
+                        if (disableNice)
```

Comment:
Yes i checked in different variants like disableNice is undefined, disableNice = true and disableNice = false - all is working.